app/helpers/report_helper.rb

```diff
@@ -146,4 +146,122 @@ def reports_group_title
          :group_description => User.current_user.current_group.description}
     end
   end
+
+  def dashboard_widget_summary(widget, sb_data)
+    schedule = widget.miq_schedule
+    timezone = schedule&.run_at ? schedule.run_at[:tz] : session[:user_tz]
+    safe_join([
+                widget_basic_information(widget),
+                widget_status(widget, sb_data, timezone),
+                widget_sb_type_report(widget, sb_data),
+                widget_timer(schedule, timezone, sb_data),
+                widget_visibility(widget),
+              ])
+  end
+
+  private
+
+  def row_data(label, value, read_only = "")
+    {:cells => {:label => "#{read_only} #{_(label)}", :value => value}}
+  end
+
+  def widget_basic_information(widget)
+    read_only = widget.read_only ? '*' : ''
+    data = {:title => _('Basic Information'), :mode => "miq_widget_basic_info"}
+    data[:rows] = [
+      row_data('Title', widget.title, read_only),
+      row_data('Descrption', widget.description, read_only),
+      row_data('Active', widget.enabled),
+      row_data('Default', widget.read_only)
```

Comment:
This one seems to pass read_only to the second parameter instead of the third...bug?

Reply:
- The third parameter for `read_only` is optional, otherwise its just a `*` that needs to be added with the label.
- The '`Default`' label is presented with the` widget.read_only` (boolean) value as expected as seen in the description screenshots